packages/openapi-to-graphql-cli/src/openapi-to-graphql.ts

```diff
@@ -131,7 +131,10 @@ function readFile(path) {
       : yaml.safeLoad(fs.readFileSync(path, 'utf8'))
     return doc
   } catch (e) {
-    console.error('Error: failed to parse YAML/JSON')
+    console.error(
+      `Failed to parse JSON/YAML. Ensure correct file ` +
+        `extension (i.e. '.json' or '.yaml').`
```

Comment:
I am unsure about this error message: we are testing for a `.json` ending, but the error message makes it seem that in addition, only `.yaml` is supported. I have seen quite a few `.yml` files for YAML also. Now, if something goes wrong parsing them, it may seem to users the reason is that the ending is `.yml` instead of `.yaml`, which is not the problem, though.

Reply:
Do we want to make a check for all possible endings (`.json`, `.yml`, and `.yaml`)?

Reply:
Do we want to make it a more stringent test? Instead of just an regex inclusion check, make sure that the last characters of the path are exact matches of the expected extensions?